SECURITY.md

```diff
@@ -0,0 +1,242 @@
+# Caliptra Project Security Incident Response
```

Comment:
Like @jhprabhu said in the meeting, let's mark this as a `Work in Progress` before merging.

Reply:
done

Reply:
done